java-components/resource-model/src/main/java/com/redhat/hacbs/resources/model/v1alpha1/Util.java

```diff
@@ -0,0 +1,23 @@
+package com.redhat.hacbs.resources.model.v1alpha1;
+
+import com.redhat.hacbs.resources.model.v1alpha1.jbsconfigstatus.ImageRegistry;
+
+public class Util {
+    public static ImageRegistry parseRegistry(String registry) {
```

Comment:
Moved ImageRegistry parser to model so it may be used by both cache and build-request-processor